src/library/returnBook/rETURN_bOOK_cONTROL.java

```diff
@@ -6,26 +6,26 @@
 public class ReturnBookControl {// Class name changed from 'rETURN_bOOK_cONTROL' to 'ReturnBookControl'
 // indents size are changed to 4 spaces
 	private ReturnBookUI Ui;
-	private enum ControlState { INITIALISED, READY, INSPECTING };// Changed 'cOnTrOl_sTaTe' to 'ControlState'
-	private ControlState State;// Changed class name 'cOnTrOl_sTaTe sTaTe' to 'ControlState State'
+	private enum controlState { INITIALISED, READY, INSPECTING };// Changed 'cOnTrOl_sTaTe' to 'controlState'

```

Comment:
enum should shart with an uppercase letter and should be CamelBack.
'controlState' should be 'ControlState'